### ISSUE TITLE
Cannot figure out the layer toggle key? Where is it defined?

### ISSUE THREAD

**Issue by zenny** [OneDeadKey/arsenik#39]
Hi,

This repo seems like a thoroughly researched for ease. Yet I could not find exactly where the key for toggle-layers is defined? Did I miss something? Sorry for if my query is too naive. Thanks in advance.

Cheers,
/zenny

**Comment by zenny** [comment]
After going through the kanata dir, I found that Space and left alt (AltGr) is used for layer switch. But the layer seem to get back to base layer as soon as the layer-switch keys are pressed! Is there any way to remain in the chosen mode without needing to press the Space or AltGr to activate and switch back to base layer by pressing the same key or cycle through a specific key? Thanks.

**Comment by Nuclear-Squid** [comment]
Hi ! Which options did you enable in your config ? In the `kanata.kbd` file, you can find a lot of `include` statements, most of which are commented, this is where you choose your config options by (de)commenting the different `include` statements and activating parts of the layout. By default Arsenik doesn’t do much, most of the options are opt-in. You can configure the layer taps on lines 30–32

**Comment by zenny** [comment]
> Hi ! Which options did you enable in your config ? In the `kanata.kbd` file, you can find a lot of `include` statements, most of which are commented, this is where you choose your config options by (de)commenting the different `include` statements and activating parts of the layout. By default Arsenik doesn’t do much, most of the options are opt-in. You can configure the layer taps on lines 30–32

@Nuclear-Squid Thanks. My config looks like:

```
;;==========================================================================;;
;;                                                                          ;;
;;  With Arsenik, choose the features you want for your keyboard:           ;;
;;  angle mods, Vim-like navigation layer, Mac/Azerty/Qwertz support, etc.  ;;
;;                                                                          ;;
;;==========================================================================;;

;; Every section is mandatory and should enable one and only one `include`
;; - enable each feature by un-commenting the related line.
;; - a commented line starts with ;;

;; Live-reload the configuration with Space+Backspace (requires layer-taps).


;;-----------------------------------------------------------------------------
;; Original key arrangement on your keyboard: Mac or PC.
;; Choose here if you want to add an angle mod: ZXCVB are shifted to the left.
;; See https://colemakmods.github.io/ergonomic-mods/angle.html for more details.

(include defsrc_pc.kbd)  ;; PC, standard finger assignment
;; (include defsrc_mac.kbd)  ;; Mac, standard finger assignment
;; (include defsrc_pc_anglemod.kbd)  ;; PC, ZXCVB are shifted to the left
;; (include defsrc_mac_anglemod.kbd)  ;; Mac, ZXCVB are shifted to the left


;;-----------------------------------------------------------------------------
;; `Base` layer: standard or dual keys? (layer-taps, homerow mods?)
;; If you just want angle mod, you still have to enable the standard base.

;; (include deflayer_base.kbd)  ;; standard keyboard behavior
;; (include deflayer_base_lt.kbd)  ;; layer-taps on both thumb keys
(include deflayer_base_lt_hrm.kbd)  ;; layer-taps + home-row mods

;; Note: not enabling layer-taps here makes the rest of the file useless.


;;-----------------------------------------------------------------------------
;; `Symbols` layer

(include deflayer_symbols_noop.kbd)  ;; AltGr stays as-is
;; (include deflayer_symbols_lafayette.kbd)  ;; AltGr programmation layer like Ergo‑L
;; (include deflayer_symbols_noop_num.kbd)  ;; AltGr stays as-is + NumRow layers
;; (include deflayer_symbols_lafayette_num.kbd)  ;; AltGr prog layer + NumRow layers


;;-----------------------------------------------------------------------------
;; `Navigation` layer: ESDF or HJKL?

(include deflayer_navigation.kbd)  ;; ESDF on the left, NumPad on the right
;; (include deflayer_navigation_vim.kbd)  ;; HJKL + NumPad on [Space]+[Q]

;; Replace XX by the keyboard shortcut of your application launcher, if any.
;; Mapped on [Space]+[P] in both navigation layers.

;; (defalias run M-p)  ;; [Command]-[P]
(defalias run XX)  ;; do nothing


;;-----------------------------------------------------------------------------
;; Aliases for `Symbols` and `Navigation` layers
;; Depends on PC/Mac and keyboard layout

;; (include defalias_ergol_pc.kbd)  ;; Ergo‑L PC
;; (include defalias_qwerty-lafayette_pc.kbd)  ;; Qwerty‑Lafayette PC
(include defalias_qwerty_pc.kbd)  ;; Qwerty / Colemak PC
;; (include defalias_qwerty_mac.kbd)  ;; Qwerty / Colemak Mac
;; (include defalias_azerty_pc.kbd)  ;; Azerty PC
;; (include defalias_qwertz_pc.kbd)  ;; Qwertz PC
;; (include defalias_bepo_pc.kbd)  ;; Bépo PC
;; (include defalias_optimot_pc.kbd)  ;; Optimot PC

;;-----------------------------------------------------------------------------
;; Extra configuration
;; You should not modify this, only if you need to.

(defcfg
  ;; Enabled makes kanata process keys that are not defined in defsrc
  ;; Fixes altgr for Windows (see Arsenik issue #22)
  process-unmapped-keys yes
  windows-altgr cancel-lctl-press
)

;; vim: set ft=lisp
```

I am wondering how can the layer remain without continue pressing the toggle key? Merci beaucoup.
